django/db/backends/base/features.py

```diff
@@ -36,6 +36,7 @@ class BaseDatabaseFeatures:
     has_select_for_update_nowait = False
     has_select_for_update_skip_locked = False
     has_select_for_update_of = False
+    has_select_for_no_key_update = False
```

Comment:
`has_select_for_no_key_update` -> `has_select_for_update_no_key`

Reply:
I think `has_select_for_update_no_key` is a better reflection of what is happening on the database layer, which is not an "`UPDATE` + a `NO KEY` option", but a `NO KEY UPDATE` operation.
I know it looks different than the other options as is, but should it be consistent with the rest of the code or with what happens on the DB?